### ISSUE TITLE
datepicker doesn't use converter

### ISSUE THREAD

**Issue by lyahim** [TheCoder4eu/BootsFaces-OSP#288]
Hello All,

I use in backing bean LocalDate and try to inject into datepicker component.
When the page is loading the component said: Value could be either String or java.util.Date.
I use `<f:converter ... />` the component shouldn't use the converter?

Best Regards,
Mihaly

**Comment by zhedar** [comment]
In #341 I stumbled upon the Date and Time bits in the `DatePicker`. I'm a little concerned about the changes you did in https://github.com/TheCoder4eu/BootsFaces-OSP/pull/292 and hope you can shed a little light on how using Java8 in Java6 source can work.
What baffles me is, that Maven doesn't complain about that at all. I thought it should throw a compile errors at least if the source and target level is set to 1.6.
However I'm running it with JDK8, so it may just run fine. But couldn't this change break Java6 environments? Did anyone ever run the Snapshot in such an environment and did test that?

It may be convenient to be able to use the new Date and Time API directly with the component, but the standard way still seems to be to [define a custom converter](http://jj-blogger.blogspot.de/2015/06/utilizing-java-8-date-time-api-with-jsf.html) to keep the 1.6 compatibility of the framework.

**TL;DR:** We should check for 1.6 compatibility in the current snapshot and probably remove the `LocalDate` parts again.
